src/evaluation_system/api/plugin_manager.py

```diff
@@ -1041,18 +1041,14 @@ def get_history(
     QuerySet[History]
         Results from the database query.
     """
-    user = user or User()
-
-    if plugin_name is not None:
-        plugin_name = plugin_name.lower()
-
-    return user.getUserDB().getHistory(
-        plugin_name,
-        limit,
+    user_ = user or User()
+    return user_.getUserDB().getHistory(
+        tool_name=plugin_name.lower() if plugin_name else None,
```

Comment:
Please avoid one line if else clauses they make the code harder to read

Reply:
done